src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public Figure createRandomFigure() {
+        return new Circle(getRandomInt());
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead, create a separate variable and pass it on for better code readability.